requirements.txt

```diff
@@ -8,8 +8,6 @@ Pillow>=7.1.2
 PyYAML>=5.3.1
 requests>=2.23.0
 scipy>=1.4.1
-torch>=1.7.0
```

Comment:
Is this intentional?

Reply:
Yep. The torch requirements are already filled by sparseml further down in the file, so these get overwritten anyway.